source/Server/Integration/JiraRestClient.cs

```diff
@@ -134,28 +134,18 @@ HttpClient CreateHttpClient(IOctopusHttpClientFactory octopusHttpClientFactory)
 
     class JiraIssue
     {
-        public JiraIssue()
-        {
-            Fields = new JiraIssueFields();
-        }
-        
         [JsonProperty("key")]
-        public string Key { get; set; }
+        public string Key { get; set; } = string.Empty;
```

Comment:
Can these be explicitly set to null in the source JSON? If so, shouldn't it be nullable?

Reply:
The key is required, it's the issue Id in this structure